AY2122S2-CS2103T-T13-3/tp
# Implement delete module feature

## Related issues
Addresses #53

## Description

1. `mdelete` delete module feature implemented
2. Some test cases implemented

## How to test

1. Using the default data file, create a new module using the following command: `madd m/cs2105`.
2. Delete the module using `mdelete 4`. The module should be deleted successfully.
3. Attempt to delete another module with dependent meetings, e.g. `mdelete 1`. The delete operation is blocked.

## Other information
The test cases will need to be expanded to test for modules that have dependent and no dependent meetings (in v1.4).
